build/repo.targets

```diff
@@ -1,43 +0,0 @@
-<Project InitialTargets="TagCiBuilds">
-  <Target Name="TagCiBuilds" Condition=" '$(CI)' == 'true' AND '$(BUILD_REASON)' != 'PullRequest' ">
```

Comment:
We still need these tags to be generated. These are used to control our release pipeline.